lib/adapters/http/index.js

```diff
@@ -1009,7 +1009,7 @@ function HttpPouch(opts, callback) {
       headers: clone(host.headers),
       method: 'POST',
       url: genDBUrl(host, '_revs_diff'),
-      body: JSON.stringify(req)
```

Comment:
Seems that way. In most cases strings _weren't_ being passed, so these ones were out of the norm (as far as I could tell).

CouchDB was returning errors ("body must be valid JSON"), because request was seeming stringifying _again_ (so an escaped string was being sent).